superset-frontend/src/components/DatabaseSelector/index.tsx

```diff
@@ -205,6 +205,7 @@ export default function DatabaseSelector({
     [formMode, getDbList, handleError, sqlLabMode],
   );
 
+  console.log(db, schema, 'INSIDE DATABASE SELECTOR');
```

Comment:
Make sure this is deleted in the final :)

Reply:
Of course, I apologize, I was trying to find them all and evidently missed one

Reply:
I told him it'd be okay to leave the code a little messy if he's just pushing it up to a draft PR for us to help debug, that's on me 😁

Reply:
No worries! And its just in draft form so its all good. Just something to keep in mind for when it goes into review (I am guilty of this one 0_0 )